specs/windows/etw_process_events.table

```diff
@@ -0,0 +1,29 @@
+table_name("etw_process_events")
+description("Windows process execution events.")
+schema([
+    Column("type", TEXT, "Event Type"),
+    Column("pid", BIGINT, "Process ID"),
+    Column("ppid", BIGINT, "Parent Process ID"),
+    Column("session_id", INTEGER, "Session ID"),
+    Column("flags", INTEGER, "Process Flags"),
+    Column("exit_code", INTEGER, "Exit Code - Present only on ProcessStop events"),
+    Column("path", TEXT, "Path of executed binary"),
+    Column("cmdline", TEXT, "Command Line"),
+    Column("username", TEXT, "User rights - primary token username"),
+    Column("token_elevation_type", TEXT, "Primary token elevation type - Present only on ProcessStart events"),
+    Column("token_elevation_status", INTEGER, "Primary token elevation status - Present only on ProcessStart events"),
+    Column("mandatory_label", TEXT, "Primary token mandatory label sid - Present only on ProcessStart events"),
+    Column("datetime", TEXT, "Event timestamp in log UTC format"),
+    Column("time_windows", BIGINT, "Event timestamp in Windows format", hidden=True),
+    Column("time", BIGINT, "Event timestamp in Unix format", hidden=True),
```

Comment:
`timestamp` is the usual name for this

Reply:
both `bpf_process_events `and `es_process_events` use time as the column name for event timestamp. I didn't want to diverge from that so I've ended up leaving `time` instead of timestamp.

Reply:
👍